### ISSUE TITLE
QModal not closing on iOS 9.3.x

### ISSUE THREAD

**Issue by TheSinding** [quasarframework/quasar#1482]
Hello, I'm trying to use the QModal on an iOS device running 9.3.x, and the modal will open, but not close again.
example code: 
```html
    <q-modal ref="modal">
        <q-btn @click="$refs.modal.close()">Close</q-btn>
    </q-modal>
     <q-btn @click="$refs.modal.open()">Open</q-btn>
```

It **will** close on newest android, firefox, iOS etc.

I know this is properly because of the old version of iOS, just thought you should know.

Quasar: 0.14.7
OS: 
Node:
NPM:
Browsers:
iOS: 9.3.x
Android:
Any other software related to your bug:

**Comment by TheSinding** [comment]
I think something else is fucking it up, my bad

**Comment by TheSinding** [comment]
No, it's not me, it's sporadic, sometimes it works, and sometimes it doesnt

**Comment by rstoenescu** [comment]
This may happen only while developing due to HMR & window history API issues. It does not happen in production. Similar tickets have been reopened and this has been addressed in v0.15, so closing it.

**Comment by TheSinding** [comment]
Oh, okay, sorry.. I should have used the search bar..
